.eslintrc.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
   extends: [require.resolve('@backstage/cli/config/eslint')],
+  rules: {
+    'notice/notice': 'off',
+    'no-nested-ternary': 'off',
```

Comment:
maybe turn it off only for a particular line, not whole project? Frankly saying I dont like when people use nested ternary, because its not readable. Sometimes its better to use some kind of switch-case

Reply:
It's not just one case, but a common react pattern to have separate boolean flags indicating data loading state:
`const { value, loading, error } = useFunctionIds();`
And then render proper jsx:
```tsx
const projectSelect = loading ? (
    <CircularProgress />
  ) : error ? (
    <Typography>
      {`Error occured while loading available projects: ${error}`}
    </Typography>
  ) : (
    <FormControl>
    ...
```
Switch is not possible here because it's different variables and if-else looks like that:
```tsx
let projectSelect: any;
  if (loading) {
    projectSelect = <CircularProgress />;
  } else if (error) {
    projectSelect = (
      <Typography>
        {`Error occured while loading available projects: ${error}`}
      </Typography>
    );
  } else {
    projectSelect = (
      <FormControl>
```

I find the first snippet more readable.

Reply:
I don't have any personal feelings about nested ternary vs. no nested ternary. It will be easier for the Backstage community to contribute to this plugin if we use the same prettier and eslint rules as Backstage does.

The notice needs to be off or we will have Spotify copyright notices on all our files but we should strive to have as few other changes as possible.

Reply:
@mcalus3 please check this article, this is what i. meant by switch case and could be the Best solution here to avoid nested ternaries [switch(true)] 
https://medium.com/trabe/using-switch-true-in-javascript-986e8ad8ae4f

Reply:
Fixed using the switch(true) pattern :)